services/web/server/src/simcore_service_webserver/resource_manager/registry.py

```diff
@@ -71,8 +71,8 @@ def _decode_hash_key(cls, hash_key: str) -> RegistryKeyPrefixDict:
             if hash_key.endswith(f":{RESOURCE_SUFFIX}")
             else hash_key[: -len(f":{ALIVE_SUFFIX}")]
         )
-        key = dict(x.split("=") for x in tmp_key.split(":"))
-        return RegistryKeyPrefixDict(**key)
+        key = dict(part.split("=") for part in tmp_key.split(":"))
+        return RegistryKeyPrefixDict(**key)  # type: ignore
```

Comment:
Q: why is this one not liked?

Reply:
RegistryKeyPrefixDict is not total but at the same time he does not like **key